code-challenges.test.js

```diff
@@ -21,10 +21,29 @@
 // Example input: 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
-
+describe('fibSequence', ()=>{
+  it('returns array of Fibonacci Sequence if greater than two', ()=>{
+    expect(fibSequence(number > 2)).toEqual([1, 1, 2, 3, 5, 8])
+    expect(fibSequence(number < 2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+  })
+})
+//GOT RED
 
 // b) Create the function that makes the test pass.
 
+// create a function that takes in a number > 2 as an argument.
+// evaluate the input to break it down into a sequence
+// return an array of the numbers in the sequence
+// I got stuck with this one, but this was my line of thinking
```

Comment:
It's a  good start. I think you would be helped by getting all the notes down first. Maybe even writing out a test case in pseudocode.